scripts/add_electricity.py

```diff
@@ -87,19 +87,27 @@
 
 import logging
 import os
+import sys
 
 import geopandas as gpd
 import numpy as np
 import pandas as pd
 import powerplantmatching as pm
 import pypsa
 import xarray as xr
-from _helpers import configure_logging, read_csv_nafix, update_p_nom_max
+from _helpers import (
+    configure_logging,
+    create_logger,
+    handle_exception,
+    read_csv_nafix,
+    update_p_nom_max,
+)
 from powerplantmatching.export import map_country_bus
 
 idx = pd.IndexSlice
 
-logger = logging.getLogger(__name__)
+create_logger(__name__)
+sys.excepthook = handle_exception
```

Comment:
Just seen; can't we place this line as well into create_logger?

Reply:
It appears, we perfectly can with the current logger definition. Great :D